### PR TITLE
Frontier - Add Galaxy

### PR DESCRIPTION
**Pull request by jayjay-crypto**

### CODE REVIEW

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -1670,6 +1670,35 @@
...
+      "name": "Glx",
+      "display": "glx",
+      "symbol": "GLX",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-236",
+        "source_denom": "uglx"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/glx.png",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1699]: Please point this to Chain registry
https://github.com/cosmos/chain-registry/blob/master/galaxy/images/glx.png
and
https://github.com/cosmos/chain-registry/blob/master/galaxy/images/glx.svg

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 2]
```diff
@@ -1670,6 +1670,35 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/hash.svg"
       },
       "coingecko_id": "provenance-blockchain"
+    },
+    {
+      "description": "GLX is the staking token of the Galaxy Chain",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1675]: Which asset description is correct? If this, can the chain registry be corrected?
https://github.com/cosmos/chain-registry/blob/master/galaxy/assetlist.json

**Comment by JeremyParish69** [comment]
Please see corequisite for token listing on Osmosis zone, and register Galaxy under SLIP173
https://docs.osmosis.zone/integrate/frontend.html#how-to-add-an-asset-onto-the-osmosis-assets-page
https://docs.osmosis.zone/integrate/registration.html#how-to-register-a-bech32-prefix-onto-slip173

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 1 above)
Comment (jayjay-crypto) [reply osmosis-1/osmosis-frontier.assetlist.json]: I have make change to chain registry

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 2 above)
Comment (jayjay-crypto) [reply osmosis-1/osmosis-frontier.assetlist.json]: Yes chain registry more corrected

Comment (JeremyParish69) [reply osmosis-1/osmosis-frontier.assetlist.json]: yep, this new description looks good. Please propose correction on Chain Registry

**Comment by jayjay-crypto** [comment]
I have changed my PR based on your recommendation.
PR to SLIP173 already merdge https://github.com/satoshilabs/slips/pull/1314 . 
Thank for your help.

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 2 above)
Comment (jayjay-crypto) [reply osmosis-1/osmosis-frontier.assetlist.json:1675]: I push a PR to correct chain registry. I will keep you update

**Comment by JeremyParish69** [comment]
Thanks for the changes. Pending PR for token listing to Frontier and addition of liquidity onto Osmosis.

**Comment by jayjay-crypto** [comment]
> Please see corequisite for token listing on Osmosis zone, and register Galaxy under SLIP173 https://docs.osmosis.zone/integrate/frontend.html#how-to-add-an-asset-onto-the-osmosis-assets-page https://docs.osmosis.zone/integrate/registration.html#how-to-register-a-bech32-prefix-onto-slip173


PR to SLIP173 already merdge https://github.com/satoshilabs/slips/pull/1314 .
Thank for your help.

**Comment by jinhopkg** [comment]
> Thanks for the changes. Pending PR for token listing to Frontier and addition of liquidity onto Osmosis.

hello.
Aded osmosis gamm pool 696.:)

thanks!

**Comment by JeremyParish69** [comment]
Can we please upload the glx.png to the local images folder as well?

**Comment by jayjay-crypto** [comment]
> Can we please upload the glx.png to the local images folder as well?

Shouldn’t we point to the local png more than the chain registry?

**Pull request merged**